app/Util.php

```diff
@@ -40,6 +40,13 @@ public static function setEnvVariables($root = '/') {
 				die($e->getMessage());
 			}
 		}
+		else if ( file_exists($root . '/.test.env') && is_readable($root . '/.test.env') ) {
+			try {
```

Comment:
I think I will update this file to remove `try catch` block as `file_get_contents` won't throw an exception.